python/librmm/pyproject.toml

```
@@ -55,7 +55,7 @@ requires = [
 [tool.scikit-build]
 build-dir = "build/{wheel_tag}"
 cmake.build-type = "Release"
-cmake.minimum-version = "3.26.4"
+cmake.version = "CMakeLists.txt"
```

Great. This PR tackles (1) and (2) from https://github.com/rapidsai/build-planning/issues/58. Should we do (3) as well?

> set a minimum-version (minimum scikit-build-core version) in the [tool.scikit-build-core] table in pyproject.toml

```suggestion
cmake.version = "CMakeLists.txt"
minimum-version = "0.10"
```

---

+1, was just about to say the same thing. If we're going to have to touch all these configs, might as well also close https://github.com/rapidsai/build-planning/issues/58 in the process.